### ISSUE TITLE
Username override

### ISSUE THREAD

**Issue by Kladki** [rosebudmods/skin-overrides#4]
### description

Change username shown in tab, nametags, and possibly chat, but the latter will likely be more complicated.

**Comment by orifu** [comment]
this mod is really geared around skins, so if you want to have nicknames i would recommend mods like [styled nicknames](https://modrinth.com/mod/styled-nicknames) or [drogstyle](https://modrinth.com/mod/drogstyle)

**Comment by orifu** [comment]
you're right about the nickname mods being server side... i don't think this would be hard to implement, so i might add this in the future
